portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
+    height: 100%;
+    width: 100%;
 }
+ 
+body{
+    height: 100%;
+    margin: 0;
+    background: linear-gradient(45deg,
+      rgba(20, 18, 19, 0.9) 30%,
+      rgba(0, 94, 105, 0.9) 100%),
+    url(Images/github_pattern.svg);
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+}
+ 
+#content {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+}
+ 
+#title{
+    text-align: center;
+}
+ 
+h1{
+    color: white;
+}
+ 
+#text{
+    background: rgba(255,255,255,0.7); 
+    display: inline-block;
+}
+
+#image-container{
```

Comment:
Perhaps a better name for this would be self-container? It's generally better to keep names very specific (so it's clear what it's for) or very generic (if it can be reused). If you plan on reusing this, feel free to keep it as image-container!

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming

Reply:
Done!